jetbrains-core/src/software/aws/toolkits/jetbrains/ui/wizard/python/SamInitDirectoryBasedSettingsPanel.java

```diff
@@ -41,6 +41,9 @@ public JPanel getComponent() {
         return mainPanel;
     }
 
+    @NotNull
+    public JTextField getSamExecutableField() { return samExecutableField; }
```

Comment:
I hadn't noticed this before, but we follow a different pattern for all the other 'panels' where we expose the fields themselves rather than having to create getters for them. See [`EditFunctionPanel`](https://github.com/aws/aws-toolkit-jetbrains/blob/master/jetbrains-core/src/software/aws/toolkits/jetbrains/services/lambda/upload/EditFunctionPanel.java).

Reply:
Sure, I can change this.